app/components/cases/submissions/decisionmaking-flow-selector.hbs

```diff
@@ -1,47 +1,67 @@
 <AuFormRow ...attributes>
   <AuLabel @required={{true}} @inline={{true}}>{{capitalize (t "case")}}</AuLabel>
-  <PowerSelect
-    @disabled={{@disableEdit}}
-    @search={{perform this.searchDecisionmakingFlows}}
-    @searchMessage={{t "search-on-case-name"}}
-    @searchEnabled={{true}}
-    @noMatchesMessage={{t "no-results-found"}}
-    @loadingMessage={{t "please-be-patient"}}
-    @selected={{this.decisionmakingFlowOrTitle}}
-    @onChange={{this.onSelectDecisionmakingFlow}}
-    class="au-u-1-1"
-    as |decisionmakingFlow|
-  >
-    {{or decisionmakingFlow.case.shortTitle decisionmakingFlow}}
-  </PowerSelect>
-  {{#unless @disableEdit}}
-    <AuButton
-      @skin="link"
-      @icon="add"
-      class="au-u-margin-top-small"
-      {{on "click" (queue (toggle "isAddingNewDecisionmakingFlow" this) this.setFocus)}}
-    >
-      {{t "add-new-case"}}
-    </AuButton>
-  {{/unless}}
+  <div class="au-c-inline-input-button">
+    {{#if @selectedDecisionmakingFlow}}
+      <AuInput
+        value={{@selectedDecisionmakingFlow.case.shortTitle}}
+        @disabled={{true}}
+      />
+      {{#unless @disableEdit}}
+        <AuButton
+          @skin="secondary"
+          @icon="pencil"
+          {{on "click" (queue this.onClearSelectedDecisionmakingFlow this.setFocus)}}
+        >
+          {{t "new-case"}}
+        </AuButton>
+        <AuButton
+          @skin="secondary"
+          @icon="add"
+          {{on "click" (queue (toggle "isUsingExistingDecisionmakingFlow" this) this.setFocus)}}
```

Comment:
`this.setFocus` does nothing with the current changes (when opening the confirmationModal).
It used to point to the input field inside the confirmationModal, but now it points to the new case input field while you are opening a modal with a powerselect. So setting focus to a field behind a modal is not needed.

I added the focus to reduce clicks. I don't think we can open the trigger of the powerselect so you can start typing right away. So only focusing when you press "nieuw dossier" seems ok. you can start typing right away.